apps/src/netsim/NetSimRouterLogModal.js

```diff
@@ -467,6 +535,7 @@ NetSimRouterLogModal.prototype.canSetRouterLogMode_ = function () {
  */
 NetSimRouterLogModal.prototype.setRouterLogMode_ = function (mode) {
   this.isAllRouterLogMode_ = mode === 'all';
+  usingNewLogBrowser() && this.render();
```

Comment:
That's a good question.  My original thought was that we're going to do a bunch of manual management of the log rows anyway (since I'm really just replacing the .ejs template with .jsx, not the whole `NetSimRouterLogModal` class) and it felt a little funny to hook up redux for this bit when we'd have to manually trigger renders in other places.

On the other hand I've been thinking about how we more or less reinvented redux in `NetSimTable`/`NetSimShard`, and _could_ overhaul all of NetSim... but if it ain't broke...